### PR TITLE
Add N&N, blog for 4.15.0.AM1

### PR DESCRIPTION
**Pull request by jeffmaury**
Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

File: blog/4.15.0.am1.adoc [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+==== Wildfly 19 Server Adapter
+
+A server adapter has been added to work with Wildfly 19. It adds support for
+Java EE 8, Jakarta EE 8 and Microprofile 3.3.
+
+Please note that while creating a Wildfly 19 server adapter, you may get a warning.
+This is going to be fixed for the release. 
+
+related_jira::JBIDE-27092[]
```
Comment (sbouchet) [inline blog/4.15.0.am1.adoc:153]: typo fix
```suggestion
related_jira:[JBIDE-27092]
```

File: documentation/whatsnew/server/server-news-4.15.0.AM1.adoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+== Wildfly 19 Server Adapter
+
+A server adapter has been added to work with Wildfly 19. It adds support for
+Java EE 8, Jakarta EE 8 and Microprofile 3.3.
+
+Please note that while creating a Wildfly 19 server adapter, you may get a warning.
+This is going to be fixed for the release. 
+
+related_jira::JBIDE-27092[]
```
Comment (sbouchet) [inline documentation/whatsnew/server/server-news-4.15.0.AM1.adoc:16]: typo fix
```suggestion
related_jira:[JBIDE-27092]
```

File: documentation/whatsnew/server/server-news-4.15.0.AM1.adoc [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+== EAP 7.3 Server Adapter
+
+The server adapter has been adapted to work with EAP 7.3.
+
+related_jira::JBIDE-27143[]
```
Comment (sbouchet) [inline documentation/whatsnew/server/server-news-4.15.0.AM1.adoc:22]: typo fix
```suggestion
related_jira:[JBIDE-27143]
```

File: blog/4.15.0.am1.adoc (hunk 1 above)
Comment (jeffmaury) [reply blog/4.15.0.am1.adoc:153]: That's was the format we always used.

**Pull request merged**